src/app/components/BibPage/BibPage.jsx

```diff
@@ -61,22 +60,21 @@ const BibPage = (props) => {
               aria-relevant="additions removals"
               aria-describedby="results-description"
             >
-              <BibMainInfo bib={bib} />
-            </div>
-
-            <div className="nypl-column-three-quarters">
               <div className="nypl-item-details">
-                <ItemHoldings
-                  shortenItems={shortenItems}
-                  items={items}
-                  bibId={bibId}
-                  title="AVAILABILITY"
-                />
-                <MarcRecord bNumber={bNumber[0]} />
-                <BibDetails
-                  bib={bib}
-                  title="Bib details"
-                />
+                <h1>{title}</h1>
+                <dl>
```

Comment:
I prefer it here buuuut some of them need to conveniently wrap, React-style, so I am updating accordingly and removing this line.